src/component/tooltip/helper.ts

```diff
@@ -27,3 +31,40 @@ export function shouldTooltipConfine(tooltipModel: Model<TooltipOption>): boolea
         // In richText mode, the outside part can not be visible.
         : tooltipModel.get('renderMode') === 'richText';
 }
+
+function testStyle(styleProps: string[]): string | undefined {
+    if (!env.domSupported) {
+        return;
```

Comment:
I am not sure that if here `undefined` returned, 
then `TRANSFORM_VENDOR` and `TRANSITION_VENDOR` will become `undefined`, 
then 
```js
const CSS_TRANSITION_VENDOR = toCSSVendorPrefix(TRANSITION_VENDOR, 'transition');
const CSS_TRANSFORM_VENDOR = toCSSVendorPrefix(TRANSFORM_VENDOR, 'transform');
```
An `undefined` will input to `toCSSVendorPrefix`.
And then a null pointer error will be thrown in `toCSSVendorPrefix`?

Reply:
Sorry. You are right. I missed it.